speech/src/main/java/com/example/speech/TranscribeStreamingV2.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.speech;
+
+// [START speech_to_text_transcribe_streaming_v2]
+
+import com.google.api.gax.rpc.BidiStream;
+import com.google.cloud.speech.v2.*;
+import com.google.protobuf.ByteString;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class TranscribeStreamingV2 {
+    private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
```

Comment:
issue: pass project ID into the sample function. Look at other samples for how to do this.